packages/shared/lib/utils/error.manager.ts

```diff
@@ -107,6 +108,19 @@ class ErrorManager {
         });
 
         logger.error(`Exception caught: ${JSON.stringify(e, Object.getOwnPropertyNames(e))}`);
+
+        if (e instanceof Error && tracer) {
+            // Log to datadog manually
+            // https://github.com/DataDog/dd-trace-js/issues/1944
+            const span = tracer.scope().active();
+            if (span) {
+                span.addTags({
+                    'error.msg': e.message,
+                    'error.stack': e.stack,
+                    'error.type': e.name
+                });
```

Comment:
you can simply do `span.setTag('error', e)` which is taking care of splitting into those 3 tags https://docs.datadoghq.com/tracing/trace_collection/custom_instrumentation/dd_libraries/nodejs/?tab=errors